Client/Client.cc

```diff
@@ -158,6 +172,17 @@ TimeoutException::TimeoutException(const std::string& error)
 {
 }
 
+ConfigurationExceptionBad::ConfigurationExceptionBad(const std::string& error)
+    : Exception(error)
+{
+}
+
+ConfigurationExceptionChanged
+::ConfigurationExceptionChanged(const std::string& error)
```

Comment:
half done. i agree with the break after (. I'm cool with a strict coding standard but I have been unable to figure out a general rule in the logcabin style to deal with number of indents post (\n. We can delve into this if you feel it's important.